### PR TITLE
"Module Manager" tab should be the first tab in Modules Tab

### LINKED ISSUE #11692: Generally, the "Module Manager" tab should be the first tab here in Modules Tab
**PrestaShop Version**
V1.7.5.0 RC1

**Is your feature request related to a problem? Please describe.**
See my screenshot, All clear in the screenshot

**Describe the solution you'd like**
Just reverse the order
download 21

### PR DESCRIPTION
**Pull request by rblaurin**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | "Module Manager" tab should be the first tab in Modules Tab
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #11692
| How to test?  | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12850)

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Hello,

would you mind to rebase your pull request please? We have a conflict on the file 1.7.6.0.sql :(

Thanks

File: install-dev/upgrade/sql/1.7.6.0.sql [hunk 1]
```diff
@@ -22,3 +22,7 @@ ALTER TABLE `PREFIX_admin_filter`
   DROP INDEX IF EXISTS `search_idx`,
 	ADD UNIQUE INDEX `admin_filter_search_id_idx` (`employee`, `shop`, `controller`, `action`, `filter_id`)
 ;
+
+/* Module Manager tab should be the first tab in Modules Tab */
+UPDATE `PREFIX__tab` SET `position` = 0 WHERE `class_name` = 'AdminModulesSf' AND `position`= 1;
```
Comment (Matt75) [inline install-dev/upgrade/sql/1.7.6.0.sql]: `PREFIX__tab` should not be `PREFIX_tab` instead  ?

File: install-dev/upgrade/sql/1.7.6.0.sql [hunk 2]
```diff
@@ -22,3 +22,7 @@ ALTER TABLE `PREFIX_admin_filter`
...
+UPDATE `PREFIX__tab` SET `position` = 1 WHERE `class_name` = 'AdminParentModulesCatalog' AND `position`= 0;
```
Comment (Matt75) [inline install-dev/upgrade/sql/1.7.6.0.sql]: `PREFIX__tab` should not be `PREFIX_tab` instead ?

**Review by Matt75 — COMMENTED** [review]
Be careful with PREFIX

File: install-dev/upgrade/sql/1.7.6.0.sql (hunk 2 above)
Comment (rblaurin) [reply install-dev/upgrade/sql/1.7.6.0.sql]: Yes, my bad, sorry

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @rblaurin

**Pull request merged**